src/main/java/seedu/address/logic/commands/AddAliasCommand.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Represents a command to add an alias for a specified term.
+ * An alias is a shorthand representation for a longer term.
+ */
+public class AddAliasCommand extends Command {
+
+    public static final String COMMAND_WORD = "alias";
+
+    public static final String MESSAGE_SUCCESS = "New Alias added: %1$s";
+
+    private final String alias;
+    private final String toReplace;
+
+    /**
+     * Creates an AddCommand to add the specified {@code Person}
+     */
+    public AddAliasCommand(String alias, String toReplace) {
```

Comment:
This is a very much needed addition - it significantly enhances the functionality for our target users, especially masters of CLI